firebase-dev.json

```diff
@@ -1,10 +1,23 @@
 {
-  "hosting": {
-    "public": "dist",
-    "ignore": [
-      "firebase.json",
-      "**/.*",
-      "**/node_modules/**"
-    ]
-  }
+  "hosting":
+    {
+      "site": "yfthig-dev",
+      "public": "dist",
+      "ignore": ["**/devops/**", "**/.*", "**/node_modules/**"],
+      "headers" : [ {
+        "source": "**",
+        "headers" : [
+          { "key" : "Access-Control-Allow-Origin", "value" : "*" },
+          { "key" : "X-Frame-Options", "value" : "DENY" },
+          { "key" : "X-Content-Type-Options", "value" : "nosniff" },
+          { "key" : "X-XSS-Protection", "value" : "1; mode=block" },
+          {
+            "key": "Content-Security-Policy",
+            "value": "default-src 'self'; frame-src 'self' *.bcros.ca *.gov.bc.ca *.hotjar.com; script-src 'self' 'unsafe-eval' 'unsafe-inline' *.bcros.ca *.gov.bc.ca *.hotjar.com *.googleapis.com; style-src 'self' 'unsafe-inline' *.cloudflare.com; font-src 'self' *.bcros.ca *.gov.bc.ca *.hotjar.com *.cloudflare.com; img-src 'self' data:; connect-src 'self' *.bcros.ca *.gov.bc.ca *.launchdarkly.com *.hotjar.com *.postescanada-canadapost.ca *.sentry.io wss://*.hotjar.com *.hotjar.io; manifest-src 'self';"},
```

Comment:
Add Hotjar to img-src ([like this](https://github.com/bcgov/sbc-auth/commit/05753ec61666d6001def014d74f7953c84dee851)) so that Hotjar icons are displayed.

Reply:
Also in prod and test files below.

Reply:
Thanks.

Did you mean to remove the `*.bcros.ca` domains from some of the sources?

Reply:
Ah, OK. Resolving.